client/src/components/Card.js

```diff
@@ -2,15 +2,29 @@ import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 
 Card.propTypes = {
-  children: PropTypes.node,
+  text: PropTypes.string,
+  author: PropTypes.string,
 }
 
-export default function Card({ children }) {
-  return <Wrapper>{children}</Wrapper>
+export default function Card({ text, author }) {
+  return (
+    <Wrapper>
+      <p>{text}</p>
+      <AuthorText>
+        - {author.length > 20 ? author.substr(0, 19) + '...' : author}
```

Comment:
It might be a better choice to do this with css. Also: there is an "ellipsis" character.